attalos/imgtxt_algorithms/main.ipynb

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
Hm, I believe that these iPyNB's are primarily what our visitors have been using to quickly get up to speed. (In fact, they have only been working with the notebooks.) While running python scripts are, indeed, necessary for results reproducibility, it is necessary to have the iPyNBs to play around with. Let's keep the notebooks, please.